apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -372,8 +372,9 @@ const EventTypePage = (props: EventTypeSetupProps) => {
       throw new Error(t("seats_and_no_show_fee_error"));
     }
 
+    const { availability, ...rest } = input;
     updateMutation.mutate({
```

Comment:
As a followup, we should either ignore these warnings explicitly, or better we should do something like {availability:_availability, ...rest}